docs/software/installing/python/cray_python.md

```diff
@@ -0,0 +1,120 @@
+# Installing and using packages with Cray Python
+
+The HPE Cray distribution provides Python 3 along with some of the most
+common packages used for scientific computation and data analysis. These
+include:
+
+  - numpy and scipy - built using GCC against HPE Cray LibSci
+  - mpi4py - built using GCC against HPE Cray MPICH
+  - dask
+
+The HPE Cray Python distribution can be loaded (either on the front-end
+or in a submission script) using:
+
+    module load LUMI/21.12  # First load the LUMI software stack
+    module load cray-python # Python version depends on LUMI module loaded
+
+!!! tip
+    The HPE Cray Python distribution provides Python 3. There is no Python 2
+    version as Python 2 is now deprecated.
+
+!!! tip
+    The HPE Cray Python distribution is built using GCC compilers. If you wish
+    to compile your own Python, C/C++ or Fortran code to use with HPE Cray
+    Python, you should ensure that you compile using `PrgEnv-gnu` to make sure
+    they are compatible.
+
+## Adding your own packages
+
+If the packages you require are not included in the HPE Cray Python
+distribution, further packages can be added using `pip`.  We recommend either
+installing them globally for your user account, as described in this section,
+or into virtual environments, see [next section](## Setting up virtual
```

Comment:
another broken internal link

Reply:
Ah, thanks for checking. Did you discover by hand or does mkdocs warn you about them? I got no warnings for those two.

Reply:
I test all the links, by hands :)